recipes/mf2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mf2" %}
+{% set version = "2020.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 43e0be769160e19459d6cf7adb98373f8d6042bf4ca804049bc1aba58333851c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - setuptools
```

Comment:
do you really need ``setuptools`` to run this project?
```suggestion
```

Reply:
I'm defining the version number only in `setup.py`, and then importing it in `__init__.py` using `pkg_resources` from `setuptools`.
See https://github.com/sjvrijn/mf2/blob/master/mf2/__init__.py#L28